CHANGELOG.md

```diff
@@ -26,8 +26,9 @@
 
 ## Fixed Issues
 
-- [core] Re-enable the circuit breakers for the destination and xsuaa services.
+- [core] Re-enable the circuit breakers for the destination and XSUAA services.
 - [odata] Fix encoding of filter strings.
+- [core] Add or overwrite the content length header for CSRF fetch requests. As those are always `HEAD` requests, the content length is always set to 0.
```

Comment:
```suggestion
- [core] Add or overwrite the `Content-Length` header for CSRF fetch requests. As those are always `HEAD` requests, the content length is always set to 0.
```

Reply:
I wanted to avoid writing out the case, because it is case independent.